seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/sink/HttpSink.java

```diff
@@ -81,11 +70,12 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
                                             entry -> String.valueOf(entry.getValue().unwrapped()),
                                             (v1, v2) -> v2)));
         }
+        this.seaTunnelRowType = rowType;
     }
 
     @Override
-    public void setTypeInfo(SeaTunnelRowType seaTunnelRowType) {
-        this.seaTunnelRowType = seaTunnelRowType;
+    public String getPluginName() {
+        return "Http";
```

Comment:
I think you can use `CONNECTOR_IDENTITY` to replace `Http` now.

Reply:
done